examples/3Scaling/2ParallelDatabase/pDB14_AdjBFSTEST.m

```diff
@@ -23,13 +23,13 @@
 
 if DoDB
     DBsetup;                          % Create binding to database.
-    G = DBaddJavaOps('edu.mit.ll.graphulo.MatlabGraphulo','instance','localhost:2181','root','secret');
+    %G = DBaddJavaOps('edu.mit.ll.graphulo.MatlabGraphulo','instance','localhost:2181','root','secret');
     
     rnameMat = [getName(Tadj) '_Mat_k' num2str(kmax)];
     TresMat = DB(rnameMat);
     deleteForce(TresMat);
     TresMat = DB(rnameMat);
-    G.addAllSumCombiner(rnameMat);
+    DB.addAllSumCombiner(rnameMat);
```

Comment:
Are you sure this method works? I can't find the source for `addAllSumCombiner`.